assets/styles/layouts/_Header.scss

```diff
@@ -327,6 +327,13 @@
 				justify-content: flex-start;
 				align-items: flex-start;
 
+				.button-more-integrations {
+
+					a {
+						color: #ababab !important;
```

Comment:
It should be even smaller if you check the LiveAgent website.

Reply:
changed in last commit